templates/__common__/config/tasks/unused-css.js

```diff
@@ -0,0 +1,68 @@
+// native
+const path = require('path');
+
+// packages
+const fs = require('fs-extra');
+const { PurgeCSS } = require('purgecss');
+const CleanCSS = require('clean-css');
+
+// internal
+const paths = require('../paths');
+const { isProductionEnv } = require('../env');
+const { logErrorMessage, replaceExtension } = require('../utils');
+
+// output locations
+const stylesFolder = isProductionEnv ? paths.appDistStyles : paths.appTmpStyles;
+const htmlFolder = isProductionEnv ? paths.appDist : paths.appTmp;
+
+const parseCSS = async () => {
+  let result = [];
+  try {
+    // run purgeCSS
+    result = await new PurgeCSS().purge({
+      content: [
+        `${htmlFolder}/**/*.html`,
+        `${paths.appSrc}/scripts/components/**/*.js`,
+      ],
+      css: [`${stylesFolder}/*.css`],
+    });
+  } catch (error) {
+    logErrorMessage(error);
```

Comment:
Should this error be thrown in the catch, instead of logged? I noticed on `npm run build` that the build still completes successfully even if there is the error.

Reply:
Such a good call! Just fixed that.